### PR TITLE
chore(docs): Update tutorial documentation part 4, 7 & 8

### PR DESCRIPTION
**Pull request by rubengmurray**
## Description

I recently went through all 8 steps of the Gatsby tutorial pages and thought the documentation was fantastic. But I did come across a very small number of lines that I thought with a tweak could have saved me a couple of minutes, so have made them here and added them in as a PR.

They are pretty simple:

- Made the localhost:9000 a link in Part-Eight (which I found super useful in the early parts)
- Made a dummy .png Icon for Part-Eight to enable people to run through the tutorial if they don't have a 512x512 png handy.
- Amended some code block highlighting in part-four, where two lines differ from previously mentioned code in the tutorial, but aren't highlighted.
- Added a restart the development server note to part-seven, where it wasn't obvious it was needed.
- Amended contents of the part-eight code blocks + highlighting to make it feel like the parts are joined together, given they all point to the requirements needed for Lighthouse & PWA's.

### CODE REVIEW

File: docs/tutorial/part-eight/index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ gatsby build
 gatsby serve
 ```
 
-Once this starts, you can now view your site at `localhost:9000`.
+Once this starts, you can view your site at http://localhost:9000.
```
Comment (LekoArts) [inline docs/tutorial/part-eight/index.md]:
```suggestion
Once this starts, you can view your site at [`localhost:9000`](http://localhost:9000).
```

I'd actually prefer this format, what do you think?

File: docs/tutorial/part-eight/index.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Quoting [Google](https://developers.google.com/web/fundamentals/web-app-manifest
 npm install --save gatsby-plugin-manifest
 ```
 
-2. Add a favicon for your app under `src/images/icon.png`. The icon is necessary to build all images for the manifest. For more information, look at the docs for [`gatsby-plugin-manifest`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/README.md).
+2. Add a favicon for your app under `src/images/icon.png`. For the purposes of this tutorial you can use [this](./icon.png), should you not have one available. The icon is necessary to build all images for the manifest. For more information, look at the docs for [`gatsby-plugin-manifest`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/README.md).
```
Comment (LekoArts) [inline docs/tutorial/part-eight/index.md]:
```suggestion
2. Add a favicon for your app under `src/images/icon.png`. For the purposes of this tutorial you can use [this example icon](./icon.png), should you not have one available. The icon is necessary to build all images for the manifest. For more information, look at the docs for [`gatsby-plugin-manifest`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/README.md).
```

I'd prefer to use this image here: https://github.com/gatsbyjs/gatsby-starter-default/blob/master/src/images/gatsby-icon.png

**Comment by marcysutton** [comment]
@rubengmurray most of the usage I've seen uses backticks `` for strings rather than quotes, as they allow for multiline strings and interpolation. I'd say start with those wherever possible and then go for single quotes.

**Comment by rubengmurray** [comment]
I've updated the double-quotes to backticks as per @marcysutton comment, in line with other references to `gatsby-config.js` in the tutorial pages e.g. https://www.gatsbyjs.org/tutorial/part-four/#data-in-gatsby

**Comment by marcysutton** [comment]
Thanks so much for your work on this! :taco:

**Pull request merged**